packages/protocol/migrationsConfig.js

```diff
@@ -125,6 +125,7 @@ const DefaultConfig = {
       adjustmentSpeed: 0.1,
     },
     membershipHistoryLength: 60,
+    commissionUpdateDelay: 30000,
```

Comment:
Consider a number that corresponds to a integer number of days. Suggest 3
```suggestion
    commissionUpdateDelay: 51840,  // Approximately 3 days
```